### PR TITLE
Index page and styling.

### PR DESCRIPTION
**Pull request by sinansevgi**
This is the first Responsive Web Design Project of Microverse HTML and CSS Curriculum

    Our design has three breakpoints.
    - < 768px
    - between 768x and 1024px
    - > 1024px

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello Team!! Great work I can see the effort put into it!!

:100: Descriptive enough README
:100: Descriptive Pull Request
:100: Descriptive Commit Messages
:100: Great to add a live preview link to the README
:100: Great to add a preview screenshot to the README
:100: Loved the content Table in the CSS
:100: Nice to have a sticky navbar

## Required Changes
### [Mandatory] :red_circle: 

- [x] LinkedIn link is not displaying properly in README
image

- [x] Website is missing a meaningful ```<title>```. See inline comment.

- [x] Missing 2 space indentation for ```<body>```  and inside elements. See inline comment.

- [x] Write a descriptive alternative text on ```alt=""```, don't leave it blank. See inline comment.

- [x] Website has a horizontal scrolling issue. Might be related to the next issue

- [x] Navbar is not displaying as the original.
image

- [x] Style does not match original
image

- [x] Navbar doesn't match original in tablet view
image

- [x] Latest funding section is not displaying as original
image

- [x] navbar not displaying properly in mobile view
image

- [x] Right margin wider than original
image

- [x] Lates news section not displaying properly on mobile view
image

- [x] Latest funding and other section's section are not displaying properly in mobile view
image

- [x] Latest deals is not displaying properly on mobile view
image

- [x] Footer not displaying as original in mobile view
image

## [OPTIONAL] :large_blue_circle: 

- [x] Use meaningful names for the branch in future projects.
**Actual**:
```index-page```
**Improved** :+1: :
```thenextweb-clone```

- [x] Be more descriptive in the commit messages in future projects. This makes the history of the project more readable and lets the reader or programmers understand the decisions made.
**Actual**:
```worked on the main section and it's content```
**Improved** :+1: :
```Add Latest Funding section```


### [Comments] :green_circle: 
- To avoid this kind of commits, make a habit to run the local linters before committing any changes. This will help maintain a better history with meaningful commit messages and without any noise.
image

- Use the **browser media query inspector** to see if the website is looking as expected in different screen-sizes / browser viewports. 


## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+- GitHub: [@wintan1418](https://github.com/wintan1418)
+- Twitter: [@juwonoluwadare](https://twitter.com/oluwadarejuwon)
+- LinkedIn: [Oluwadare Juwon](https://www.linkedin.com/in/oluwadare-juwon-048a391a8/)
+
+
+👤 **Sinan Sevgi**
+
+- GitHub: [@sinansevgi](https://github.com/sinansevgi)
+- LinkedIn: (https://www.linkedin.com/in/sinan-s-52559437/)
+- Twitter: [@nativeofcyber1](https://twitter.com/nativeofcyber1)
```
Comment (jubaan) [inline README.md]: Linkedin is missing the Link Text. --> [@sinansevgi]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,741 @@
...
+
+<head>
+  <meta charset="UTF-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <meta content="ie=edge" http-equiv="X-UA-Compatible">
+  <link href="css/style.css" rel="stylesheet" type="text/css">
+  <link crossorigin="anonymous" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+        integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog=="
+        rel="stylesheet"/>
+  <title>Document</title>
```
Comment (jubaan) [inline index.html]: Use a descriptive title for the clone

File: index.html [hunk 3]
```diff
@@ -0,0 +1,741 @@
...
+  function myFunction() {
+    if (window.pageYOffset >= sticky) {
+      navbar.classList.add("sticky")
+    } else {
+      navbar.classList.remove("sticky");
+    }
+  }
+</script>
+</body>
+</html>
```
Comment (jubaan) [inline index.html]: 2 space indentation missing for ```<body>``` and inside elements.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,741 @@
...
+      <li><a href="#">EVENTS</a></li>
+      <li><a href="#">ABOUT</a></li>
+      <li><a href="#">ADVERTISE</a></li>
+      <li><a href="#">JOBS</a></li>
+      <li><a href="#">CONTACT</a></li></ul>
+  </div>
+  <div class="legal">
+    <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="">
+    &copy; 2006–2020 The Next Web B.V. Made with &lt;3 in Amsterdam.
+    <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/stackpath.svg" alt="">
```
Comment (jubaan) [inline index.html]: This is just an example but there are other images that are missing a description on the alternative text attribute. 
**Actual**
``````
** Improved** :+1: :
```Robert De Niro```

Having and alternate text help SEO and Screen-readers

**Review by jubaan — CHANGES_REQUESTED** [review]

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

Great work team!

## Required Changes
### [Mandatory] :red_circle: 

- [x] Horizontal scrolling still appears. Read Comments

- [x] Irregular spaces between elements
image

- [x] Letter size doesn't change accordingly to screen size
image

- [x] Footer still not showing as original
image

- [x] Latest Deals not showing as original
image

- [x] Navbar not displaying as original in tablet mode

- [x] Navbar is missing search icon

image

### [Comments] :green_circle: 
- The website still has a scrolling horizontal issue, this is not present on Mozilla Firefox but it is in Other browsers like Opera and Chrome. This could be related to a vendor prefix in the CSS, Try to use autoprfixer online and evaluate with another browser, to see if it shows as intended.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,1331 @@
...
+  }
+}
+
+@media screen and (max-width: 1024px) and (min-width: 769px) {
+  body {
+    font-size: 16px;
+  }
+
+  .tnw-logo-visible {
+    display: block !important;
```
Comment (umairarshadbutt) [inline css/style.css]: remove `!important`

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @wintan1418 and @sinansevgi
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly work on the suggested changes  👇.
- [x] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Latest funding section is not displaying as original (As mentioned previous TSE. This issue is not fixed on medium screen)
- [x] Irregular spaces between elements (Funding section)

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — APPROVED** [review]
## Congratulations Approved 🎉
-----------------------------------------------------------------------------------------------------------------------------
Hello Developer 👋
You are on 🔥
Thank you so much for making those changes and I am very happy to approve the project. You have done a great job. 👏
Please keep working hard and move forward with your curriculum.
Looking forward to seeing your next project. 😄
---------------------------------------------------------------------------------------------------------------
**To highlight**
- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

- [x] **Optional** Please make your PR descriptive. 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**